e2e-tests/insurance/cypress/e2e/journeys/export-contract/check-your-answers/check-your-answers-summary-list/check-your-answers-summary-list-contracted-awarded-other-method.spec.js

```diff
@@ -0,0 +1,42 @@
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import { assertMinimalExportContractSummaryListRows } from '../../../../../../../shared-test-assertions';
+import FIELD_IDS from '../../../../../../../constants/field-ids/insurance/export-contract';
+import application from '../../../../../../../fixtures/application';
+
+const { ROOT, EXPORT_CONTRACT } = INSURANCE_ROUTES;
+
+const {
+  HOW_WAS_THE_CONTRACT_AWARDED: { OTHER_AWARD_METHOD },
+} = FIELD_IDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(
+  'Insurance - Export contract - Check your answers - Summary list - application under total contract value threshold, no private insurance attempt, not using an agent, contracted awarded other method',
```

Comment:
I don't think we need to mention that it's under the threshold, only if it's over

Similararily we don't need so say e.g "not using an agent", "not using loss payee" etc etc

Reply:
Same with the other flags I think it should just be 

```ts
`Insurance - Export contract - Check your answers - Summary list - contract awarded with ${OTHER_METHOD}`
```